tests/queries.ts

```diff
@@ -42,5 +48,95 @@ testClient(async function binaryType() {
 
   assertEquals(row[0], expectedBytes);
 
-  await CLIENT.query("INSERT INTO bytes VALUES($1);", expectedBytes);
+  await CLIENT.query(
+    "INSERT INTO bytes VALUES($1);",
+    { args: [expectedBytes] },
+  );
+});
+
+// MultiQueries
+
+testClient(async function multiQueryWithOne() {
+  const result = await CLIENT.multiQuery("SELECT * from bytes;");
+  const row = result[0].rows[0];
+
+  const expectedBytes = new Uint8Array([102, 111, 111, 0, 128, 92, 255]);
+
+  assertEquals(row[0], expectedBytes);
+
+  await CLIENT.multiQuery([{
+    text: "INSERT INTO bytes VALUES($1);",
+    config: { args: [expectedBytes] },
+  }]);
+});
+
+testClient(async function multiQueryWithManyString() {
+  const result = await CLIENT.multiQuery(
+    "SELECT * from bytes;SELECT * FROM timestamps; SELECT * FROM ids;",
+  );
+  assertEquals(result.length, 3);
+
+  const expectedBytes = new Uint8Array([102, 111, 111, 0, 128, 92, 255]);
+
+  assertEquals(result[0].rows[0][0], expectedBytes);
+
+  const expectedDate = Date.UTC(2019, 1, 10, 6, 0, 40, 5);
+
+  assertEquals(
+    result[1].rows[0][0].toUTCString(),
+    new Date(expectedDate).toUTCString(),
+  );
+
+  assertEquals(result[2].rows.length, 2);
+
+  await CLIENT.multiQuery([{
+    text: "INSERT INTO bytes VALUES($1);",
+    config: { args: [expectedBytes] },
```

Comment:
Why is `config` needed? I think `{ text, args }` interface was sufficient for this method

Reply:
I might have misunderstood what you meant with removing `encoder` and `name`, I thought you mean from only these functions, should I prune all the methods from this (or the whole module even)?

Reply:
No, I mean that I don't understand why `QueryConfig { text, args }` was replaced with `QueryType { text, config: { args } }` - it seems they essentially do the same thing - if so then probably no need to change `QueryConfig` at all

Reply:
I thought it might be cleaner to have a config input where all the settings/args etc. can be inside so that it is future proof for future settings/options. Do you think it's better to keep it as it is?

Reply:
Yeah I think so - `QueryConfig` can still be extended in the future with additional fields

Reply:
ok, then i will revert it

Reply:
@bartlomieju It should be all good now 👍